modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -120,35 +120,38 @@ ifndef::ibm-z-kvm[]
 ----
 rd.neednet=1 \
 console=ttysclp0 \
-coreos.inst.install_dev=/dev/dasda \ <1>
+coreos.inst.install_dev=/dev/dasda \// <1>
 ignition.firstboot ignition.platform.id=metal \
-coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \
-coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
+coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <2>
+coreos.inst.ignition_url=http://<HTTP_server>/master.ign \// <3>
 ip=10.19.17.2::10.19.17.1:255.255.255.0::enbdd0:none nameserver=10.19.17.1 \
-zfcp.allow_lun_scan=0 \ <2>
+zfcp.allow_lun_scan=0 \// <4>
 rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
-rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000 \ <3>
-zfcp.allow_lun_scan=0 \
-rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
-rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000
+rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000 \// <5>
 ----
 <1> For installations on DASD-type disks, add `coreos.inst.install_dev=/dev/dasda`. Omit this value for FCP-type disks.
-<2> For installations on FCP-type disks, add `zfcp.allow_lun_scan=0`. Omit this value for DASD-type disks.
-<3> For installations on DASD-type disks, replace with `rd.dasd=0.0.3490` to specify the DASD device.
+<2> Specify the location of the `rootfs` artifact for the `kernel` and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<3> Specify the location of the Ignition config file. Use  `master.ign` or `worker.ign`. Only HTTP and HTTPS protocols are supported.
+<4> For installations on FCP-type disks, add `zfcp.allow_lun_scan=0`. Omit this value for DASD-type disks.
```

Comment:
allow_lun_scan=<value>
disables the automatic LUN scan for FCP devices that run in NPIV mode if set to 0, n, or N. To enable the LUN scanning set the parameter to 1, y, or Y. When the LUN scan is disabled, all LUNs must be configured through the unit_add zfcp attribute in sysfs. LUN scan is enabled by default.

It is not required to omit the "zfcp.allow_lun_scan" parameter, if you boot from a dasd disk. 
Removing the line "zfcp.allow_lun_scan=0" will basically add the line "zfcp.allow_lun_scan=1", because "allow_lun_scan" is enabled by default.

The above statement is written in many other parts of the documentation.  We can correct this later with a new PR
Therefore I approve to this PR for now.